rpcs3/display_sleep_control.cpp

```diff
@@ -4,6 +4,10 @@
 #include <windows.h>
 
 #elif defined(__APPLE__)
+#if defined(__clang__)
+#pragma clang diagnostic ignored "-Wold-style-cast"
```

Comment:
I think `You're Holding It Wrong`.
https://clang.llvm.org/docs/UsersManual.html#controlling-diagnostics-via-pragmas
You need to `push`, then `ignore` and finally `pop` before `#elif defined(HAVE_QTDBUS)` line